LaTeX/LaTeX.sublime-syntax

```diff
@@ -198,6 +207,123 @@ contexts:
     - include: paragraph-pop
     - include: else-pop
 
+  xparse-argspec:
+    - match: (?=\{)
+      set:
+        # deliberately using an anonymous scope here. we need two match the `{`
+        # in two stages, so we can apply the `clear_scopes`, and still be able
+        # to push `xparse-argspec` onto the stack without messing up scoping.
+        - clear_scopes: 1
+        - match: '\{'
+          scope: punctuation.definition.group.brace.begin.latex
+          set: xparse-argspec-content
+
+    - include: paragraph-pop
+    - include: else-pop
+
+  xparse-argspec-content:
+    - clear_scopes: 1
+    - meta_scope: meta.function.parameters.latex
+    - include: brace-group-end
```

Comment:
I wouldn't change stacking strategy of a single contexts for very good reasons. Assuming `pragraph-pop` is not required within braced groups _(as it doesn't appear in others)_ we should keep pushing here.

```suggestion
  xparse-argspec:
    - match: \{
      scope: punctuation.definition.group.brace.begin.latex
      push: xparse-argspec-content
    - include: paragraph-pop
    - include: else-pop

  xparse-argspec-content:
    - clear_scopes: 1
    - meta_scope: meta.function.parameters.latex
    - match: \}
      scope: punctuation.definition.group.brace.end.latex
      pop: 2
```

Reply:
Pushed a proposal to handle it via `set`